### ISSUE TITLE
Shield TV no work after update 0.7.25.

### ISSUE THREAD

**Issue by THEDARKK** [schwabe/ics-openvpn#1370]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android Version Android TV 9
2. Android Vendor/Custom ROM NVIDIA
3. Device Shield TV
4. Version of the app (version number/play store version/self-built) 0.7.25
 
# Description of the issue
Hello, After updating the app today, it no longer works. The old version works on another Shield TV.
Here is a photo of the error.

![20211005_123259](https://user-images.githubusercontent.com/5897899/136008453-c2ddceb0-b7c2-4184-b2f2-39e16530d18e.jpg)

Thank you

**Comment by Quikfix73** [comment]
We have now 3 different people complaining about the same issue and reported this so my question is when can we anticipate a fix for this?

**Comment by rlopez2005** [comment]
Same problem here! (Mibox-4) There is a way to "downgrade" the app until problem is fixed?
Thanks.

**Comment by DesmondH0** [comment]
I was using OpenVPN with my Sony TV.

To workaround for a downgrade, I had done the followings:
- Uninstall the app from PlayStore
- Download the .apk file from [release](https://github.com/schwabe/ics-openvpn/releases) page (0.7.24)
- Re-import the .ovpn config

This would be a temp workaround for me, please make sure you got the .ovpn file before you uninstall the app

**Comment by bahal523252** [comment]
A general problem created after updating the software for Android TV please fix the problem defective version 07.25.0
![openvpn](https://user-images.githubusercontent.com/92033394/136205200-50e13d08-0ba2-4880-97e1-60816bc10e9f.jpeg)

**Comment by Quikfix73** [comment]
Version 0.7.27 is now available and the issue is now resolved

**Comment by dochertp** [comment]
I logged an issue (1388) which was correctly assigned as a duplicate.  I would try again with the new 0.7.27 release, but I cannot load a OPVN profile.  Possibly a cause is that my device (a Sony television) updated from Android v9 to v10 today, and may have screwed up permission.  I added this as a comment on issue 1336, as it feels like it could be the same.

Many thanks for your hard work on this.
